src/ParticleDistributions/ParticleDistributions.jl

```diff
@@ -634,4 +635,52 @@ function integrate_SimpsonEvenFast(n_bins::Int, dx::FT, y::F) where {FT <: Real,
         (17 * (y(1) + y(e)) + 59 * (y(2) + y(e - 1)) + 43 * (y(3) + y(e - 2)) + 49 * (y(4) + y(e - 3))) / 48
     return dx * retval
 end
+
+
+"""
+  compute_threshold(pdists; percentile)
+  `pdists` - tuple of particle size distributions
+  `percentile` - mass percentile
+Returns a tuple of new integral thresholds, one for each pdist, computed using the given percentile
```

Comment:
It's good to add a couple of sentences how you compute these thresholds mathematically.

Reply:
Good idea, just added